csrank/tests/test_ranking.py

```diff
@@ -55,24 +76,35 @@ def tunable_parameters(cls):
 
 @pytest.fixture(scope="module")
 def trivial_ranking_problem():
-    rand = np.random.RandomState(123)
-    x = rand.randn(100, 5, 1)
+    random_state = np.random.RandomState(123)
+    x = random_state.randn(200, 5, 1)
     y_true = x.argsort(axis=1).argsort(axis=1).squeeze(axis=-1)
     return x, y_true
 
 
 def test_fate_object_ranker_fixed(trivial_ranking_problem):
+    tf.set_random_seed(0)
+    os.environ["KERAS_BACKEND"] = "tensorflow"
+    np.random.seed(123)
+    for ranker_name in object_rankers.keys():
+        loss = 0.0
+        rtol = 1e-2
+        atol = 1e-8
+        assert object_ranker_fixed(trivial_ranking_problem, ranker_name=ranker_name, loss=loss, rtol=rtol, atol=atol)
+
+
+def object_ranker_fixed(trivial_ranking_problem, ranker_name=FATE_RANKER, loss=0.0, rtol=1e-2,
+                        atol=1e-8):
     x, y = trivial_ranking_problem
-    fate = FATEObjectRanker(n_object_features=1,
-                            n_hidden_joint_layers=1,
-                            n_hidden_set_layers=1,
-                            n_hidden_joint_units=5,
-                            n_hidden_set_units=5,
-                            kernel_regularizer=l2(1e-4),
-                            optimizer=SGD(lr=1e-3, momentum=0.9, nesterov=True))
-    fate.fit(x, y, epochs=50, validation_split=0, verbose=False)
-    pred = fate.predict(x)
-    assert np.all(pred == y)
+    ranker_params = object_rankers_params[ranker_name]
+    ranker_params['n_object_features'] = ranker_params['n_features'] = 1
+    ranker_params['n_objects'] = 5
+    ranker = object_rankers[ranker_name](**ranker_params)
+    ranker.fit(x, y, epochs=50, validation_split=0, verbose=False)
+    pred_scores = ranker.predict_scores(x)
+    pred_loss = zero_one_rank_loss_for_scores_ties_np(y, pred_scores)
+    print("ranker : {} and 0/1 pred loss: {}".format(ranker_name, pred_loss))
```

Comment:
We should not use print statements in tests.

Reply:
Removed